src/Sarif.FunctionalTests/ConverterTestData/StaticDriverVerifier/cancelspinlock_bug1.tt.sarif

```diff
@@ -229,7 +229,7 @@
                   "kind": "assignment",
                   "importance": "unimportant",
                   "properties": {
-                    "currentDataValues": "^&sdv_ControllerIrp=8520",
+                    "currentDataValues": "^\u0026sdv_ControllerIrp=8520",
```

Comment:
This is valid JSON -- `\u0026` is indeed the `"&"` character, and any character _may_ be escaped in a JSON string -- but this character does not _have_ to be escaped. Do you have any idea why this crept in? If I go further down the review, will I find that we upgraded to a new version of Newtonsoft.Json that does this conversion for some reason?